autovalet_gazebo/config/husky_gazebo_lot.rviz

```diff
@@ -438,6 +438,6 @@ Window Geometry:
     collapsed: false
   Views:
     collapsed: false
-  Width: 1600
-  X: 0
+  Width: 928
+  X: 992
   Y: 24
```

Comment:
NIT: just want to start a discussion on whether we should push RVIZ changes to `master` every time we modified it according to our own needs

Reply:
should I keep this change or undo it?
I don't feel too strongly towards either decision.

Reply:
@kob51  @SubramanianKrish